src/output-json.c

```diff
@@ -150,6 +150,9 @@ static enum JsonOutput json_out = ALERT_FILE;
 
 static enum JsonFormat format = COMPACT;
 
+/* Prepend @cee: cookie to eve JSON syslog output for CEE/Lumberjack compatibility */
+static int syslog_cee_format = FALSE; /* Default OFF */
```

Comment:
What you said makes sense. I was actually wondering about the case with multiple instances of json output, but ultimately I decided to use the global variable to follow what the surrounding code was doing.

Would you like me to open a new pull request for this feature when @regit finishes his refactor?